src/Rojo.ts

```diff
@@ -224,6 +224,18 @@ export class Rojo<C extends object = {}> extends vscode.Disposable {
       })
     )
 
+    this.server.on(
+      "exit",
+      // Code can possibly be "null", for example if the SIGTERM signal is sent
+      (code: number | null, _signal) => {
+        // Check if exited with a non-zero exit code (i.e. an error occured)
+        if (code && code !== 0) {
+          this.outputChannel.show()
```

Comment:
It looks like to me that the code already shows the output channel when anything is printed to stderr:

https://github.com/rojo-rbx/vscode-rojo/blob/4940ab4be3811b330f3c1b88e1e0b171c60493a1/src/Rojo.ts#L216

So this should be an indication that it crashed. Is this not showing for you? It does for me, when I force a crash in my testing.

Reply:
Hm, I may well have missed this somehow - odd. I can't remember exactly, but I'll get back to you

Reply:
You're right, it does show the output channel when crashed! I'm confused on how I missed that, I probably wasn't paying attention properly.

I guess this PR does add a small QoL update re: the button, but it seems even less necessary now - I understand if its too minor, feel free to close if you believe so